### ISSUE TITLE
Adaptive Dithering Not Handling Vertical Bars

### ISSUE THREAD

**Issue by HappehLemons** [MiSTer-devel/Genesis_MiSTer#194]
For example, Aladdin uses this technique for shadows:
https://imgur.com/mW512pW

This should translate to a smooth shadow.

Is this something that adaptive dithering can be updated to handle? I know dithering filters in Retroarch as expected. and the full dithering just makes games so blurry I didn't know if this could be avoided.

**Comment by sorgelig** [comment]
adaptive dithering only applied when alternating colors also include transparency. If it's simple color alternating then adaptive won't work which i'm sure is your case.

**Comment by HappehLemons** [comment]
That's correct, I wonder if it is possible to account for this? Retroarch has a solution with their gadapt filter that works very well for these scenarios. Either way, thanks for taking the time to explain.

**Comment by sorgelig** [comment]
Retroarch is software emulator. It has CPU and GPU for massive math calc. FPGA has none of that.
